src/_ert_forward_model_runner/job.py

```diff
@@ -166,9 +226,17 @@ def ensure_file_handles_closed():
         exited_message = Exited(self, exit_code)
 
         if exit_code != 0:
-            yield exited_message.with_error(
-                f"Process exited with status code {exit_code}"
-            )
+            if killed_by_oom(fm_step_pids):
```

Comment:
shall we call this `killed_by_oom` regardless the exit_code; ie. when sub-process was killed by oom?

Reply:
I think not, it should not really be a realistic scenario to have OOM intervening without something blowing up.